src/opnsense/www/js/widgets/Monit.js

```diff
@@ -54,7 +54,7 @@ export default class Monit extends BaseTableWidget {
         };
 
         this.statusColors = {
-            "1": "text-danger",
+            "0": "text-success",
             "2": "text-warning",
             "3": "text-warning"
```

Comment:
@AdSchellevis this mapping looks over-engineered from the start, but factually the fix looks good

Reply:
@fichtner to be honest, I think the problem actually starts with the api (https://github.com/opnsense/core/blob/master/src/opnsense/mvc/app/controllers/OPNsense/Monit/Api/StatusController.php). It would be better if the endpoint would also send a readable status back, in which case you could just use it without further conversion (only prefix `text-`) .

Reply:
Improving the API would also be ok, though keeping it minimal/compatible could be challenging